app/server.js

```diff
@@ -5,14 +5,118 @@ const swaggerUi = require('swagger-ui-express');
 const YAML = require('yamljs');
 const swaggerDocument = YAML.load('./swagger.yaml'); // Replace './swagger.yaml' with the path to your Swagger file
 const app = express();
-
+const PORT = process.env.PORT || 3000;
+const cors = require('cors');
 app.use(bodyParser.json());
+app.use(cors());
 
 // Importing the data from JSON files
 const users = require('../initial-data/users.json');
 const brands = require('../initial-data/brands.json');
 const products = require('../initial-data/products.json');
 
+app.get('/api/products', (req, res) => {
+	res.status(200).json(products);
+});
+
+app.get('/api/products/:id', (req, res) => {
+  const product = products.find(p => p.id === req.params.id);
```

Comment:
no need to be cheap, keep it readable
```suggestion
  const product = products.find(product => product.id === req.params.id);
```